install-dev/upgrade/php/ps_1760_copy_data_from_currency_to_currency_lang.php

```diff
@@ -62,6 +62,17 @@ function ps_1760_copy_data_from_currency_to_currency_lang()
 
 function refreshLocalizedCurrencyData(Currency $currency, array $languages, LocaleRepository $localeRepoCLDR)
 {
+    $cldrLocale = $localeRepoCLDR->getLocale(Context::getContext()->language->locale);
```

Comment:
I think we should avoir relying on the context in upgrade Since we already have the list of languages and this data isn't related to a specific Language anyway you could use the first one from the array

```suggestion
    $language = new Language($languages[0]['id_lang']);
    $cldrLocale = $localeRepoCLDR->getLocale($language->locale);
```

Reply:
Nice catch!